### ISSUE TITLE
Update packaging script for geodiff 1.0

### ISSUE THREAD

**Issue by wonder-sk** [lutraconsulting/qgis-mergin-plugin#261]
Now that new geodiff does not statically link sqlite anymore, we need to update packaging for mergin plugin:
- linux
   ```bash
   patchelf --replace-needed libsqlite3-d9e27dab.so.0.8.6 libsqlite3.so.0.8.6 libpygeodiff-1.0.0-python.so
   patchelf --remove-rpath libpygeodiff-1.0.0-python.so
   ```
- windows - nothing needed
- mac os - ?

**Comment by PeterPetrik** [comment]
`9e27dab` in the name of sqlite3 lib may be mangling on geodiff wheel side... maybe we should disable mangling at all ?


macOS: 
```
Install_name_tool -add_rpath <rpath> <library_path_on_disk> 
install_name_tool -delete_rpath <rpath> <library_path_on_disk>
 install_name_tool -change <old> <new> <library_path_on_disk>
```
list linked:
`otool -L <library_path_on_disk>`

**Comment by wonder-sk** [comment]
Fixed in #269, #270, #271, #272
